packages/ra-core/src/controller/list/useUnselectAll.ts

```diff
@@ -10,11 +10,22 @@ import { useRecordSelection } from './useRecordSelection';
  * const unselectAll = useUnselectAll('posts');
  * unselectAll();
  */
-export const useUnselectAll = (resource?: string) => {
+export const useUnselectAll = ({
+    resource,
+    storeKey,
+}: {
+    resource?: string;
+    storeKey?: string;
+}) => {
```

Comment:
Unfortunately this is a BC. Can you pass the `storeKey` as optional 2nd parameter instead?